simapp/go.mod

```diff
@@ -183,13 +184,13 @@ require (
 )
 
 replace (
+	cosmossdk.io/collections => ../collections
```

Comment:
Could you remove all the collections replace for all go mod and use a pseudo version based on the latest commit from main?

Reply:
Will do in a separate PR